projects/Mallard/src/components/article/types/article.tsx

```diff
@@ -53,23 +50,15 @@ const ArticleWebview = ({
     const [height, setHeight] = useState(Dimensions.get('window').height)
     const [, { pillar }] = useArticle()
 
-    const html = useMemo(
-        () => render(article, { pillar, features, wrapLayout }),
-        [wrapLayout.width], // eslint-disable-line react-hooks/exhaustive-deps
-    )
+    const html = render(article, { pillar, features, wrapLayout })
```

Comment:
What was the reason for removing `useMemo` here?

Reply:
the list of dependencies was getting too big and the article html was needing a rerender at almost every component rerender anyway so as you said not a lot of optimisation and it made maintaining this terser

Reply:
👍